docs/content/docs/connectors/pipeline-connectors/oceanbase.md

```diff
@@ -0,0 +1,343 @@
+weight: 7
+type: docs
+aliases:
+- /connectors/pipeline-connectors/oceanbase
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# OceanBase Connector
+
+OceanBase connector can be used as the *Data Sink* of the pipeline, and write data to [OceanBase](https://github.com/oceanbase/oceanbase). This document describes how to set up the OceanBase connector.
+
+## What can the connector do?
+* Create table automatically if not exist
+* Schema change synchronization
+* Data synchronization
+
+## Example
+
+The pipeline for reading data from MySQL and sink to OceanBase can be defined as follows:
+
+```yaml
+source:
+  type: mysql
+  hostname: mysql
+  port: 3306
+  username: mysqluser
+  password: mysqlpw
+  tables: mysql_2_oceanbase_test_17l13vc.\.*
+  server-id: 5400-5404
+  server-time-zone: UTC
+
+sink:
+  type: oceanbase
+  url: jdbc:mysql://oceanbase:2881/test
+  username: root@test
+  password:
+
+pipeline:
+  name: MySQL to OceanBase Pipeline
+  parallelism: 1
+```
+
+## Connector Options
+
+<div class="highlight">
+<table class="colwidths-auto docutils">
+    <thead>
+      <tr>
+        <th>Option</th>
+        <th>Required by Table API</th>
+        <th>Default</th>
+        <th>Type</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>url</td>
+      <td>Yes</td>
+      <td></td>
+      <td>String</td>
+      <td>JDBC url.</td>
+    </tr>
+    <tr>
+      <td>username</td>
+      <td>Yes</td>
+      <td></td>
+      <td>String</td>
+      <td>The username.</td>
+    </tr>
+    <tr>
+      <td>password</td>
+      <td>Yes</td>
+      <td></td>
+      <td>String</td>
+      <td>The password.</td>
+    </tr>
+    <tr>
+      <td>schema-name</td>
+      <td>Yes</td>
+      <td></td>
+      <td>String</td>
+      <td>The schema name or database name.</td>
+    </tr>
+    <tr>
+      <td>table-name</td>
+      <td>Yes</td>
+      <td></td>
+      <td>String</td>
+      <td>The table name.</td>
+    </tr>
+    <tr>
+      <td>driver-class-name</td>
+      <td>No</td>
+      <td>com.mysql.cj.jdbc.Driver</td>
+      <td>String</td>
+      <td>The driver class name, use 'com.mysql.cj.jdbc.Driver' by default. If other value is set, you need to introduce the driver manually.</td>
```

Comment:
The MySQL driver is also excluded in flink cdc jar files, so any driver needs to be introduced manually.

Reply:
Great, I fixed it.